servers/src/main/java/tachyon/worker/netty/NettyDataServer.java

```diff
@@ -122,9 +122,9 @@ private ServerBootstrap createBootstrapOfType(final ChannelType type) {
     final int bossThreadCount = mTachyonConf.getInt(Constants.WORKER_NETTY_BOSS_THREADS, 1);
     final int workerThreadCount = mTachyonConf.getInt(Constants.WORKER_NETTY_WORKER_THREADS, 0);
```

Comment:
I thought it was weird that it was set to 0, but I looked into it and if it is 0, Netty automatically creates #procs * 2 threads. I set the default to 0, and added comments.